### PR TITLE
Fixes: libavutil/pixfmt.h: No such file or directory (ffmpeg include dir)

### PR DESCRIPTION
**Pull request by lazToum**
fixes "libavutil/pixfmt.h: No such file or directory" in Fedora 31 (use: /usr/include/ffmpeg instead of /usr/include)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -117,11 +117,13 @@ def get_extensions():
             'nvcc': nvcc_flags,
         }
 
+    libraries = []
     if sys.platform == 'win32':
         define_macros += [('torchvision_EXPORTS', None)]
 
         extra_compile_args.setdefault('cxx', [])
         extra_compile_args['cxx'].append('/MP')
+        libraries.append('_C')
```
Comment (fmassa) [inline setup.py]: Can you explain why is this needed? The PR description mentions Fedora, while this is the windows codepath

File: setup.py [hunk 2]
```diff
@@ -132,8 +134,11 @@ def get_extensions():
     if has_ffmpeg:
         ffmpeg_bin = os.path.dirname(ffmpeg_exe)
         ffmpeg_root = os.path.dirname(ffmpeg_bin)
-        ffmpeg_include_dir = os.path.join(ffmpeg_root, 'include')
-
+        include_dir = os.path.join(ffmpeg_root, 'include')
+        ffmpeg_include_dir = include_dir
+        include_subdir = os.path.join(include_dir, 'ffmpeg')
```
Comment (fmassa) [inline setup.py]: So `libavutil` etc are under a `ffmpeg` folder?
Hum, given that we don't currently have CI for Fedora, I'm afraid that this will get broken very quickly, and we might need to find a more principled way of getting the include dirs for the ffmpeg libraries.

Do you have some ideas on how to get this in a more reliable manner?

File: setup.py (hunk 1 above)
Comment (lazToum) [reply setup.py]: You are right, it should be included in a separate PR, I included it because CI builds were failing for Windows

File: setup.py (hunk 2 above)
Comment (lazToum) [reply setup.py]: Well, I think the way /usr (or /usr/local) path is found is OK, now for the {prefix}/include dir I don't see many options. Maybe searching for the header file (from inside {prefix}/include) ? Or just include both folders ({prefix}/include and {prefix}/include/ffmpeg (if the later exists).

Comment (lazToum) [reply setup.py]: edit: I pushed an update with a dummy search for the header file, it seems to be working, and does not take a lot of time.
